lib/pdd/source.rb

```diff
@@ -37,10 +37,13 @@ def initialize(file, path)
       @path = path
     end
 
-    def match_markers(line)
-      MARKERS.map do |mkr|
-        %r{(.*(?:^|\s))#{mkr}\s+#([\w\-.:/]+)\s+(.+)}.match(line)
-      end.compact
+    def match_markers(l)
```

Comment:
@Doldrums what's wrong with `line`? Why renaming to `l`?

Reply:
we have too strict rules for the length of the string, and because of the length of the regular, had to correct